tests/unit/validators/list-test.ts

```diff
@@ -0,0 +1,101 @@
+import { ValidatorFunction } from 'ember-changeset-validations';
+import { setupTest } from 'ember-qunit';
+import { TestContext } from 'ember-test-helpers';
+import { module, test } from 'qunit';
+
+import validateList, { ListValidatorFunction } from 'ember-osf-web/validators/list';
+
+interface ValidatorTestContext extends TestContext {
+    listValidator: ListValidatorFunction;
+}
+
+function validateTrue(): ValidatorFunction {
+    return (_: string, newValue: unknown) => {
+        if (!newValue) {
+            return 'false!';
```

Comment:
If we are returning strings, would it be even more clear if the return strings were something like `'Failed Validation'` and `'Passed Validation'`??

Reply:
Validator functions (as used by ember-changeset-validators) return either an error message string or `true`. I agree 'false!' isn't the best validation message. I'll improve it.

Reply:
Forgot about that entirely here. Slightly different wording for when the validation fails works for me 👍

Reply:
How's this?